lib/cc/pull_requests.rb

```diff
@@ -1,8 +1,28 @@
 class CC::PullRequests < CC::Service
+  ABLE_TO_UPDATE_STATUS_MESSAGE = "Access token is valid"
```

Comment:
Yep, we just infer it currently, but should use the house style. Will update. In the meantime: https://github.com/codeclimate/codeclimate-services/pull/111